ikvmocr

```diff
@@ -523,10 +523,10 @@ def read_screenshot_and_dump(screenshot_filename, glyphs):
                     print(line)
 
 
-def main(screenshot_filenames, glyph_db_filename):
+def main(screenshot_filenames, glyph_db_filename, ikvmcr_data_dir):
```

Comment:
`ikvmocr` <- +`o`

Maar wat mij betreft kun je er beter `{libdir}` of `{sharedir}` van maken, of `{datadir}`. Het is tenslotte allemaal voor ikvmocr in deze file.